src/status_im/ui/components/checkbox/view.cljs

```diff
@@ -19,8 +19,10 @@
           (when on-value-change
             {:on-press #(on-value-change (not checked?))}))
    [react/view (styles/icon-check-container checked?)
-    (when checked?
-      [icons/icon :tiny-icons/tiny-check {:color colors/white}])]])
+    (if platform/desktop?
```

Comment:
why is it different for desktop?

Reply:
Desktop handles :color differently, without it icon is not visible.